ui/src/criteria/concept.tsx

```diff
@@ -511,6 +511,7 @@ function searchRequest(
       entityDataset: {
         entityVariable: "c",
         selectedAttributes: attributesForEntity(entity, columns),
+        orderByAttribute: "concept_name",
```

Comment:
Since roll up counts are in, we should order by those instead of name. That would be `person_count`. However, that doesn't work for brands so you'll need to add `orderBy` and `orderDirection` property to the `EntityConfig` in the `CriteriaConfig` and use `concept_name` for brands and `person_count` for everything else. `person_count` should be sorted in descending order.